src/css/profile/mobile/common/datetimepicker.less

```diff
@@ -0,0 +1,105 @@
+.ui-datetime-picker {
+	display: flex;
+	height: 164 * @px_base;
+	margin-left: 4%;
+	margin-right: 6%;
+	padding-top: 60 * @px_base;
```

Comment:
let's comment out this until clarified with UX

Reply:
Thanks.

Reply:
App sfter remove padding.
![obraz](https://user-images.githubusercontent.com/29534410/81928432-8636a900-95e5-11ea-9cd7-4908a2c57f17.png)

Reply:
We have answer from UX team. The padding remain.